src/inventory.h

```diff
@@ -124,10 +124,17 @@ class inventory : public visitable<inventory>
          * the player's worn items / weapon
          */
         void restack( player &p );
+        void form_from_map( const tripoint &origin, int range, player *pl, bool assign_invlet = true,
```

Comment:
This overloads seems to be unnecessary. Calling `form_from_map( tripoint, int, player * )` will chose the overload `form_from_map( const tripoint &origin, int range, const player *pl, ... )` - the `player*` is automatically converted to `const player *`.